netutils/ip.py

```diff
@@ -225,16 +227,16 @@ def netmask_to_cidr(netmask):
     raise ValueError("Subnet mask is not valid.")
 
 
-def cidr_to_netmask(cidr):
+def cidr_to_netmask(cidr: int) -> str:
     """Creates a decimal format of a CIDR value.
 
     **IPv4** only.  For IPv6, please use `cidr_to_netmaskv6`.
 
     Args:
-        cidr (int): A CIDR value.
+        cidr: A CIDR value.
 
     Returns:
-      netmask (str): Decimal format representation of CIDR value.
+        Decimal format representation of CIDR value.
```

Comment:
I see the point, but I think we should change the code to assign a variable as netmask and return it. Additionally spacing seems off

Reply:
1. There was mix of two-space and four-space for `Returns` in the docstrings, I unified this which led to this diff
2. The conversion to a netmask would be a breaking change which I would prefer not to have in this MR. Do you want me to create an issue for this?

Reply:
Removed the `netmask:` bit from both so its consistent.